uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -37,145 +50,170 @@ <h4 class="modal-title">New Document</h4>
-            <a href class="btn btn-indent required-field-indicator" ng-click="showStatusField()">Set Status</a>
-          </span>
-        </label>
-        <div class="col-xs-12 col-sm-7" ng-show="statusFieldVisible()">
-          <ui-select prevent-animations theme="civihr-ui-select" name="documentStatus" allow-clear class="required-field-indicator" ng-model="document.status_id" ng-required="true">
-            <ui-select-match prevent-animations allow-clear class="ui-select-match" placeholder="Set Status">
-              {{$select.selected.value}}
-            </ui-select-match>
-            <ui-select-choices prevent-animations class="ui-select-choices" repeat="status.key as status in cache.documentStatus.arr | filter: $select.search">
-              <div ng-bind="status.value"></div>
-            </ui-select-choices>
-          </ui-select>
-        </div>
-        <label class="col-xs-12 control-label">Details</label>
-        <div class="col-xs-12">
-          <div text-angular ta-toolbar="[['bold','italics','underline','strikeThrough','ul','ol','undo','redo','clear']]" ng-model="document.details">
+            </div>
           </div>
-        </div>
-      </div>
+          <!-- Assignee & Assignment -->
+          <div class="row">
+            <!-- Assignee -->
+            <div class="col-xs-12 col-sm-7" ng-show="!document.assignee_contact_id[0] && !showAssigneeField">
+              <span class="form-control-static">
+                <a href class="btn btn-indent" ng-click="showAssigneeField = true;">Add Assignee</a>
+              </span>
+            </div>
+            <div class="col-xs-12 col-sm-8" ng-show="document.assignee_contact_id[0] || showAssigneeField">
+              <ui-select prevent-animations name="assignee" theme="civihr-ui-select" ng-model="document.temp" on-select="addAssignee($item);">
+                <ui-select-match prevent-animations class="ui-select-match" placeholder="Start typing a name or email...">
+                  {{$select.placeholder}}
+                </ui-select-match>
+                <ui-select-choices prevent-animations class="ui-select-choices" repeat="contact.id as contact in contacts.assignee | filter: $select.search" refresh="refreshContacts($select.search, 'assignee')" refresh-delay="0">
+                  <div ng-bind="contact.label"></div>
+                  <small ng-bind="contact.description[0]"></small>
+                </ui-select-choices>
+              </ui-select>
+              <h4 ng-show="document.assignee_contact_id[0]">Assignees:</h4>
+              <ul class="list-unstyled">
+                <li ng-repeat="(key,assigneeId) in document.assignee_contact_id" >
+                  <span ng-repeat="contact in cache.contact.arrSearch">
+                    <p ng-if="contact.id == assigneeId">{{ contact.label }}
+                      <a href="javascript:void(0)">&nbsp;<i ng-click="removeAssignee(key)" class="fa fa-times" aria-hidden="true"></i></a>
```

Comment:
I am adding `ng-click` only on the cross button, which is used to remove the assginee form the list of assignees.